src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -224,6 +224,13 @@ static void uct_rdmacm_cm_handle_event_connect_response(struct rdma_cm_event *ev
 
     ucs_assert(event->id == cep->id);
 
+    cep->flags |= UCT_RDMACM_CM_EP_GOT_CONNECT;
```

Comment:
why is the got_connect flag set here even though an error might happen and the connect_cb will be invoked with the error state?
It doesnt match the doc for this flag (for case of an error)
Also, uct_rdmacm_cm_ep_error_cb() will invoke the disconnect_cb is the got_event is already set but the ep wasn't connected yet.

Reply:
@alinask  Thanks for catching this, but I think it should be fixed other way - we need internal state to handle OOO rdmacm event. sometime (1/n_kilos test iterations) I see that disconnect event is arriving before connect/response event in errorless flow.
I'm going to add another flag like `UCT_RDMACM_CM_EP_CONNECT_CB_INVOKED` to determine which CB should be invoked in error flow, wdyt?

Reply:
probably should rename UCT_RDMACM_CM_EP_GOT_CONNECT to UCT_RDMACM_CM_EP_GOT_RESPONSE